app/models/patient.rb

```diff
@@ -484,9 +484,26 @@ def jurisdiction_path
     jurisdiction&.path&.map(&:name)
   end
 
-  # Get all dependents (including self if id = responder_id) that are being monitored or in continuous exposure
+  # Get all dependents (including self if id = responder_id) that are being actively monitored, meaning:
+  # - not purged AND not closed (monitoring = true)
+  #  AND
+  #    - in continuous exposure
+  #     OR
+  #    - in isolation
+  #     OR
+  #    - within monitoring period based on LDE
+  #     OR
+  #    - within monitoring period based on creation date if no LDE specified
   def active_dependents
-    dependents.where('monitoring = ? OR continuous_exposure = ?', true, true)
+    monitoring_days_ago = ADMIN_OPTIONS['monitoring_period_days'].days.ago.beginning_of_day
+    dependents.where(purged: false, monitoring: true)
+              .where('isolation = ? OR continuous_exposure = ? OR last_date_of_exposure >= ? OR (last_date_of_exposure IS NULL AND created_at >= ?)',
```

Comment:
Is it possible to use the active record `.or()` here instead of one long string?

Reply:
We could, but it would mean a lot of code duplication because within each or block you'd have to include all of the checks that were unchanged as well. I wasn't a fan of this big string either but it seemed better than the duplication. Happy to change if folks thing otherwise or there is a better solution I may be missing.

Reply:
It might at least be worth trying, and doing an explain to see what SQL is generated. But that is a fair point, and good enough reason to not make this change.